lib/workers/repository/update/pr/changelog/github/index.ts

```diff
@@ -116,26 +132,46 @@ export async function getReleaseList(
   release: ChangeLogRelease
 ): Promise<ChangeLogNotes[]> {
   logger.trace('github.getReleaseList()');
-  // TODO #7154
-  const apiBaseUrl = project.apiBaseUrl!;
-  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
-  const repository = project.repository!;
-  const notesSourceUrl = `${ensureTrailingSlash(
-    apiBaseUrl
-  )}repos/${repository}/releases`;
-  const items = await releasesCache.getItems(
-    {
-      registryUrl: apiBaseUrl,
-      packageName: repository,
-    },
-    release
-  );
-  return items.map(({ url, id, version: tag, name, description: body }) => ({
-    url,
-    notesSourceUrl,
-    id,
-    tag,
-    name,
-    body,
-  }));
+  // istanbul ignore if
+  if (process.env.DISABLE_GITHUB_CACHE === 'true') {
```

Comment:
```suggestion
  if (process.env.RENOVATE_X_DISABLE_GITHUB_CACHE === 'true') {
```

add our experimental  prefix

https://github.com/renovatebot/renovate/blob/2df8967ae27851e79d7c665483456825a21a6e6d/docs/usage/self-hosted-experimental.md#L1